apis/types/error.go

```diff
@@ -20,24 +20,24 @@ type Error struct {
 }
 
 // Validate validates this error
-func (m *Error) Validate(formats strfmt.Registry) error {
-	return nil
+func (m *Error) Validate(formats strfmt.Registry) (err error) {
+	return err
 }
 
 // MarshalBinary interface implementation
-func (m *Error) MarshalBinary() ([]byte, error) {
+func (m *Error) MarshalBinary() (b []byte, error) {
 	if m == nil {
-		return nil, nil
+		return
 	}
 	return swag.WriteJSON(m)
 }
 
 // UnmarshalBinary interface implementation
-func (m *Error) UnmarshalBinary(b []byte) error {
+func (m *Error) UnmarshalBinary(b []byte) (err  error) {
 	var res Error
-	if err := swag.ReadJSON(b, &res); err != nil {
-		return err
+	if err = swag.ReadJSON(b, &res); err != nil {
+		return
```

Comment:
Why did you revise there? You definite a variable whose type is the error, but you didn't return error info?